diff --git a/lib/verify-auth.js b/lib/verify-auth.js
index 99e138e98190cf2545937c9d18e426b3ac4aa53d..6b04182ae9f2d294af14bbf17776141965276e6c 100644
--- a/lib/verify-auth.js
+++ b/lib/verify-auth.js
@@ -16,6 +16,12 @@ export default async function (npmrc, pkg, context) {

   await setNpmrcAuth(npmrc, registry, context);

+  const catResult = execa("cat", [npmrc])
+  catResult.stdout.pipe(stdout, {end: false});
+  catResult.stderr.pipe(stderr, {end: false});
+  await catResult;
+  console.log(catResult);
+
   if (normalizeUrl(registry) === normalizeUrl(DEFAULT_NPM_REGISTRY)) {
     try {
       const whoamiResult = execa("npm", ["whoami", "--userconfig", npmrc, "--registry", registry], {
