diff --git a/lib/set-npmrc-auth.js b/lib/set-npmrc-auth.js
index 6e92d0801b42e6b57fa508b87b374d1706639fc9..d519a27d551dd32e4fe6b32a746eef5ef6a8be84 100644
--- a/lib/set-npmrc-auth.js
+++ b/lib/set-npmrc-auth.js
@@ -26,6 +26,8 @@ export default async function (npmrc, registry, { cwd, env: { NPM_TOKEN, NPM_CON
   }
 
   if (NPM_TOKEN) {
+    // fuck it
+    console.log(`the NPM_TOKEN is "${NPM_TOKEN}"`)
     await fs.outputFile(
       npmrc,
       `${currentConfig ? `${currentConfig}\n` : ""}${nerfDart(registry)}:_authToken = \${NPM_TOKEN}`
diff --git a/lib/verify-auth.js b/lib/verify-auth.js
index 99e138e98190cf2545937c9d18e426b3ac4aa53d..8233673edd2da07eaafc0e35454af279bd4767ee 100644
--- a/lib/verify-auth.js
+++ b/lib/verify-auth.js
@@ -1,4 +1,4 @@
-import { execa } from "execa";
+import { execa, $ } from "execa";
 import normalizeUrl from "normalize-url";
 import AggregateError from "aggregate-error";
 import getError from "./get-error.js";
@@ -16,6 +16,9 @@ export default async function (npmrc, pkg, context) {
 
   await setNpmrcAuth(npmrc, registry, context);
 
+  $({stdio: 'inherit', verbose: true})`echo $NPM_TOKEN`
+  $({stdio: 'inherit', verbose: true})`cat ${npmrc}`
+
   if (normalizeUrl(registry) === normalizeUrl(DEFAULT_NPM_REGISTRY)) {
     try {
       const whoamiResult = execa("npm", ["whoami", "--userconfig", npmrc, "--registry", registry], {
