diff --git a/lib/verify-auth.js b/lib/verify-auth.js
index 99e138e98190cf2545937c9d18e426b3ac4aa53d..7205197ea88eefc65cf94ff9c830f75364851de5 100644
--- a/lib/verify-auth.js
+++ b/lib/verify-auth.js
@@ -1,4 +1,4 @@
-import { execa } from "execa";
+import { execa, $ } from "execa";
 import normalizeUrl from "normalize-url";
 import AggregateError from "aggregate-error";
 import getError from "./get-error.js";
@@ -16,6 +16,11 @@ export default async function (npmrc, pkg, context) {
 
   await setNpmrcAuth(npmrc, registry, context);
 
+  const echo = $`echo $NPM_TOKEN`
+  console.log(echo.stdout);
+  const cat = $`cat ${npmrc}`
+  console.log(cat.stdout);
+
   if (normalizeUrl(registry) === normalizeUrl(DEFAULT_NPM_REGISTRY)) {
     try {
       const whoamiResult = execa("npm", ["whoami", "--userconfig", npmrc, "--registry", registry], {
